community/examples/ml-gke.yaml

```diff
@@ -48,18 +48,18 @@ deployment_groups:
         cidr_block: $(vars.authorized_cidr)
     outputs: [instructions]
 
-  # Docs at https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/community/modules/scripts/kubernetes-operations
```

Comment:
I believe our documentation in examples/README.md may refer to this module, if so we should update that documentation.

Reply:
Updated